### ISSUE TITLE
Create a Rating emoji component

### ISSUE THREAD

**Issue by nik132-eng** [Clueless-Community/seamless-ui#708]
One need to make this component using `HTML` and `Tailwind CSS`. I would suggest to use [Tailwind Playgrounds](https://play.tailwindcss.com/) to make things faster and quicker.

Here is a reference to the component. 
![image](https://user-images.githubusercontent.com/77881638/206382215-7941959f-f9fd-4fc7-b65d-e73e9963b25e.png)

After building the component please raise a PR with a screenshot of the component and add the component in `seamless-ui/Rating/src`. 
  
If you need to use any icon please use it from [Hero Icons](https://heroicons.com/)

Good luck.

**Comment by nik132-eng** [comment]
@Rajdip019  can I work on this?

**Comment by Rajdip019** [comment]
Yeah work on it add it to the others folder. name it properly. @nik132-eng

**Comment by nik132-eng** [comment]
![image](https://user-images.githubusercontent.com/77881638/206391041-29ddee09-f649-4920-8ee9-21694496b862.png) Rating-3 ?

**Comment by nik132-eng** [comment]
Or make " Other " folder and add into it

**Comment by Rajdip019** [comment]
Yeah you can go with that. That will be better. Just give some more space between the icons and make 3 versions of it as well. As the starts. @nik132-eng

**Comment by nandita27iitp** [comment]
Please assign this to me
